app/MMB/src/list/FlightListContainer.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import {
+  graphql,
+  createRefetchContainer,
+  type RelayRefetchProp,
```

Comment:
@tbergq Where did you find this type? It is not exported from the Relay package...

Reply:
_(and therefore it's exported as `any` and you can call whatever you want on the relay property)_

Reply:
```
export type RelayRefetchProp = RelayProp & {
    refetch(
        refetchVariables:
            | RelayRuntimeTypes.Variables
            | ((fragmentVariables: RelayRuntimeTypes.Variables) => RelayRuntimeTypes.Variables),
        renderVariables?: RelayRuntimeTypes.Variables,
        callback?: (error?: Error) => void,
        options?: RefetchOptions
    ): RelayRuntimeTypes.Disposable;
};
```
This is in index.ts file from `react-realy`